### PR TITLE
 feat: migrate `the-graph/master` to `main`

### PR DESCRIPTION
**Pull request by andrewgordstewart**
To create this branch, I
- removed all files from the `graph-payments` repo, except for `.git`
- copied all files from the prepared `the-graph` commit, besides `.git`, into this repo

We ought to bump the minor version when we publish a release, so this includes a `feat` commit.

### CODE REVIEW

File: packages/payments/package.json [hunk 1]
```diff
@@ -1,51 +1,52 @@
 {
   "name": "@graphprotocol/payments",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Payment Management",
+  "description": "State Channel Payment Manager",
```
Comment (andrewgordstewart) [inline packages/payments/package.json:4]: Unintentional change:
```suggestion
"description": "Graph Network State Channel Payment Management",
```

File: packages/payments/package.json [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
   "repository": "https://github.com/statechannels/graph-payments",
-  "author": "State Channels",
+  "author": "Graph Protocol, Inc.",
```
Comment (andrewgordstewart) [inline packages/payments/package.json:6]: Unintentional change:

```suggestion
  "author": "State Channels",
```

File: packages/receipts/package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@graphprotocol/receipts",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Receipt Manager",
+  "description": "State Channel Receipt Manager",
```
Comment (andrewgordstewart) [inline packages/receipts/package.json:4]: Unintentional change
```suggestion
  "description": "Graph Network State Channel Receipt Manager",
```

File: packages/receipts/package.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "repository": "https://github.com/statechannels/graph-payments",
-  "author": "State Channels",
+  "repository": "https://github.com/graphprotocol/indexer",
+  "author": "Graph Protocol",
```
Comment (andrewgordstewart) [inline packages/receipts/package.json:8]: Unintentional change
```suggestion
  "repository": "https://github.com/statechannels/graph-payments",
  "author": "State Channels",
```

**Review by Jannis — APPROVED** [review]
Great, thanks @andrewgordstewart!

**Pull request merged**